privaterelay/settings.py

```diff
@@ -227,11 +227,17 @@
     "api.apps.ApiConfig",
 ]
 
+API_DOCS_ENABLED = config("API_DOCS_ENABLED", False, cast=bool)
+if API_DOCS_ENABLED:
+    INSTALLED_APPS += [
+        "drf_yasg",
+    ]
+
 if DEBUG:
     INSTALLED_APPS += [
         "debug_toolbar",
-        "drf_yasg",
```

Comment:
Should keep this in here so that if in `DEBUG` mode we can always get Swagger. Or change the if condition in 231 to be `if API_DOCS_ENABLED or DEBUG:` so we don't accidentally add the app twice.